stacklight_tests/tests/prometheus/base_test.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from stacklight_tests.clients.prometheus import prometheus_client
+from stacklight_tests import objects
+from stacklight_tests import utils
+
+logger = logging.getLogger(__name__)
+
+
+class BaseLMATest(object):
```

Comment:
We have the same name of class in stacklight_tests.tests.base_test.BaseLMATest, maybe BasePrometheusTest would be better?